src/RadixDlt.NetworkGateway.PostgresIntegration/LedgerExtension/EntitySchemaProcessor.cs

```diff
@@ -151,12 +173,48 @@ public void ProcessChanges()
                     EntityId = lookup.EntityId,
                     SchemaHash = entry.Key.SchemaHash.ConvertFromHex(),
```

Comment:
Question: shouldn't we check if given `entityId + schemaHash` doesn't exist already? Are we sure we do not have any duplicates in our database?